### DISCUSSION TITLE
CORS: "NetworkError when attempting to fetch resource." - Trying to create a user through Studio on Self-Hosted Docker

### DISCUSSION THREAD

**Discussion by TheLexoPlexx** [supabase/supabase#27814] [Questions] [answered]
Hi there,

 I am running the latest supabase-docker container on a VM but when trying to create a user through supabase-studio (accessible at port 8000 by default), I get an error message in the top right:
`Failed to create user: An error has occurred: NetworkError when attempting to fetch resource.`

Further investigation left me without any logs on the server side, I expected my SMTP-Settings to be faulty, since I have not set that up yet but even a correct configuration with sendgrid left me with the same error.

Today, I noticed the error might be client-sided if the server doesn't show any logs and I found multiple CORS-Errors, every time I try to register a user and the Error appears. The URL mentioned in the Error is "http://localhost:3000/auth/v1/admin/users". Which is weird, because apart from my Nextjs-Frontend, there is nothing else at localhost:3000 anyways.

I have also checked back and I had the same error on my local supabase-testing-environment at home.

The question is now:
- What configuration did I get wrong that I am unable to register any user?

and bonus-question:
- Will Auth work without a correct SMTP-Configuration? I am not planning to send E-Mails anyways for now.

**Comment by TheLexoPlexx** [comment, marked as answer]
The supabase-URL in the .env-File was wrong. I removed the port and CORS was happy.

**Reply by Cali07** [reply]
Bro this saved me, i didn't wanna use gpt because it was going to make me run in circles
